src/main/java/es/upm/etsisi/views/CLI.java

```diff
@@ -27,6 +30,9 @@ public boolean isOpen(String commandTitle) {
     public void run() {
         ErrorType error;
         String commandTitle;
+        
+        DefaultParameters defaultParameters = new DefaultParameters();
+        addDefaults(defaultParameters);
```

Comment:
The best implementation would be a this.setDefaults

Reply:
And a method that overrides the actual empty lists of elements with the generated ones (assuming they follow our actual List rules)